keyboards/40percentclub/gherkin/keymaps/stevexyz/rules.mk

```diff
@@ -0,0 +1,20 @@
+# Build Options (yes/no)
+#
+BOOTMAGIC_ENABLE = lite      # Just bootloader enabled with keys(+?)
```

Comment:
I wouldn't call it required since it's only your keymap, but I'd prefer it if all the firmware size notes from here to Line 14 were deleted (the `(+####)` bits).

Reply:
it was nice to have an idea of the various size implied but I can understand that them can also vary in time. removed

Reply:
We found a while back that they were dependent on a lot of different factors (build environment, compiler version, keyboard size, etc.) and thus, they were rarely accurate.